lib/commands/command.bash

```diff
@@ -9,7 +9,44 @@ fi
 
 # Load direnv stdlib if not already loaded
 if [ -z "$(declare -f -F watch_file)" ]; then
-  eval "$(asdf exec direnv stdlib)"
+  # We need to eval direnv stdlib, but before doing so
+  # we need to determine which direnv executable we should use.
+  #
+  # The fastest way is either an already loaded $direnv variable
+  # (as exported by direnv stdlib) or a user specified $ASDF_DIRENV_BIN.
```

Comment:
I'm a bit confused: if `$direnv` is defined, doesn't that mean that the direnv stdlib is *already* loaded?

Reply:
yep, you are right, probably I was high on coffee.